### PR TITLE
gomplate: Add version 3.6.0

### PR DESCRIPTION
**Pull request by stevehipwell**
It'd be useful to add `gomplate` to _Scoop_.

### CODE REVIEW

File: bucket/gomplate.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://github.com/hairyhenderson/gomplate",
+    "description": "A flexible commandline tool for template rendering.",
+    "license": "MIT",
+    "version": "3.6.0",
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
    "version": "3.6.0",
    "description": "Template renderer with various datasource support",
    "homepage": "https://github.com/hairyhenderson/gomplate",
    "license": "MIT",
```

File: bucket/gomplate.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_windows-amd64.exe",
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
            "url": "https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_windows-amd64.exe#/gomplate.exe",
```

File: bucket/gomplate.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "a0abc996290a1a0a6c120a03adc4f3bf6de12bdc3280f280ebaa7094570a8fe1"
+        },
+        "32bit": {
+            "url": "https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_windows-386.exe",
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
            "url": "https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_windows-386.exe#/gomplate.exe",
```

File: bucket/gomplate.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "f73627c07134b48323dbfa4ab8cf216c59a074118a033cd6108d1030d62ed940"
+        }
+    },
+    "bin": [["gomplate_windows-amd64.exe", "gomplate"]],
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
    "bin": "gomplate.exe",
```

File: bucket/gomplate.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/hairyhenderson/gomplate/releases/download/v$version/gomplate_windows-amd64.exe"
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
                "url": "https://github.com/hairyhenderson/gomplate/releases/download/v$version/gomplate_windows-amd64.exe#/gomplate.exe"
```

File: bucket/gomplate.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/hairyhenderson/gomplate/releases/download/v$version/gomplate_windows-386.exe"
```
Comment (Ash258) [inline bucket/gomplate.json]:
```suggestion
                "url": "https://github.com/hairyhenderson/gomplate/releases/download/v$version/gomplate_windows-386.exe#/gomplate.exe"
```

**Comment by stevehipwell** [comment]
@Ash258 [slim binaries ](https://docs.gomplate.ca/installing/#slim-binaries) are compressed self extracting binaries.

File: bucket/gomplate.json (hunk 3 above)
Comment (Ash258) [reply bucket/gomplate.json]: !

**Comment by stevehipwell** [comment]
@Ash258 sorry must have missed the final change requested.

**Comment by stevehipwell** [comment]
@Ash258 does the label need removing before this can be merged?

**Comment by stevehipwell** [comment]
Could someone with write access please merge this?

**Pull request merged**